frontend/src/main/components/Driver/DriverAvailabilityForm.js

```diff
@@ -0,0 +1,132 @@
+import { Button, Form } from 'react-bootstrap';
+import { useForm } from 'react-hook-form'
+import { useNavigate } from 'react-router-dom';
+
+function DriverAvailabilityForm({ initialContents, submitAction, buttonLabel = "Create" }) {
+    // Stryker disable all
+    const {
+        register,
+        formState: { errors },
+        handleSubmit,
+    } = useForm(
+        { defaultValues: initialContents || {}, }
+    );
+    // Stryker restore all
+    const navigate = useNavigate();
+
+    const testIdPrefix = "DriverAvailabilityForm";
+    return (
+        <Form onSubmit={handleSubmit(submitAction)}>
+            {initialContents && (
+                <Form.Group className="mb-3" >
+                    <Form.Label htmlFor="id">id</Form.Label>
```

Comment:
Form labels should be capitalized, lowercase is a bit unprofessional and is inconsistent with other forms in the application